### DISCUSSION TITLE
Loss of BRouter external profile

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#327] [Q&A] [answered]
I was away for 2 weeks and everything went well so far.
I noticed the following 2things negatively.
1:I have configured different profiles.
Most work with an external BRouter profile.
It often happens that this external BRouter profile is replaced by the internal driving-cruiser profile.
Often, after changing the profile, instead of the external profile saved in the profile,
the driving-cruiser-cruiser profile is active.
I have to check it every time after changing the Profile.
2:Some voice prompts are suboptimal.
e.g.: before some hairpin bends comes: "Wenden" That means: Turn Around!...not optimal.
Better maybe :"180grad Kurve" or "scharfe Kurve"....

**Comment by devemux86** [comment]
@Leeb64 thanks for the report.

Have you somehow touched the external BRouter profile file?
For example, overwrite it with a new version or edit it?

**Reply by Leeb64** [reply]
I sometimes changed some of the values ​​of the external profile, but then called it up again.
But the profile disappeared without changing the profile and was replaced by the drivin-cruiser profile.
Often after switching the profile , the driving-cruiser profile was  active.

**Comment by devemux86** [comment, marked as answer]
I made some tests:
- Open an external BRouter profile

- Switch Cruiser profiles: the BRouter profile is not lost :+1: 

- Edit or overwrite the BRouter  profile: it is still not lost :+1: 

So there could be something else going on.

---

@Leeb64
Have you started Cruiser while the BRouter profile file was not in the storage?

(in this case Cruiser reverts to the internal BRouter profile)

**Comment by devemux86** [comment]
About the u-turn instructions, let's discuss them separately here:
- https://github.com/devemux86/cruiser/discussions/328

**Comment by devemux86** [comment]
Can someone else check if the external BRouter profiles are kept correctly when we change the Cruiser profile?

**Reply by airald** [reply]
I've made same tests than you with same successful result.

**Comment by Leeb64** [comment]
I did all sorts of tests again...
This time everything worked as it should...
strange....

**Reply by devemux86** [reply]
Maybe it was what I described [above](https://github.com/devemux86/cruiser/discussions/327#discussioncomment-7094627)?

When we start Cruiser and the external BRouter profile file is not in its expected location,
Cruiser reverts to the internal BRouter profile so the routing can work.
